install/installer.class.php

```diff
@@ -136,7 +101,12 @@ public function install() {
       $this->createRootEntityConfig();
       $this->createDisplayPreferences();
 
-      Config::setConfigurationValues('flyvemdm', ['version' => PLUGIN_FLYVEMDM_VERSION]);
+      Config::setConfigurationValues(
+         'flyvemdm', [
+            'version' => PLUGIN_FLYVEMDM_VERSION,
+            'schema_version' => PLUGIN_FLYVEMDM_SCHEMA_VERSION,
+         ]
+      );
```

Comment:
at the end of an installation we must update the version and schema version of the plugin